pkg/resourceinterpreter/default/native/prune/prune.go

```diff
@@ -98,9 +98,9 @@ func removeGenerateSelectorOfJob(workload *unstructured.Unstructured) error {
 		return err
 	}
 	if exist {
-		if util.GetLabelValue(matchLabels, "controller-uid") != "" {
-			delete(matchLabels, "controller-uid")
-		}
+		delete(matchLabels, "controller-uid")
```

Comment:
How about using the key word in the Kubernetes? Same as below.

Reply:
Currently kubernetes does not publish the const of controller-uid and job-name, so they cannot be used. 
Updated batchv1.ControllerUidLabel and batchv1.JobNameLabel.